app/src/app/[lng]/authorize/page.tsx

```diff
@@ -0,0 +1,29 @@
+import crypto from 'crypto';
+import AuthorizeForm from './AuthorizeForm';
+import { Auth } from "@/lib/auth";
+import { hasFeatureFlag, FeatureFlags } from '@/util/feature-flags';
+
+export default async function AuthorizePage({ params }: { params: any }) {
+
+  // XXX: Fix i18n here
+  if (!hasFeatureFlag(FeatureFlags.OAUTH_ENABLED)) {
+    return <div>{"OAuth 2.0 not enabled"}</div>
+  }
```

Comment:
i18n

Reply:
LOL, OK. I thought my XXX comment would be enough.